Source/BaseClasses/GDataServiceBase.h

```diff
@@ -21,8 +21,8 @@
   #define GDataUploadFetcherClass GTMSessionUploadFetcher
   #define GDataUploadFetcherClassStr @"GTMSessionUploadFetcher"
 
-  #define kGDataFetcherStatusUnauthorized kGTMSessionFetcherStatusUnauthorized
-  #define kGDataFetcherStatusForbidden kGTMSessionFetcherStatusForbidden
+  #define kGDataFetcherStatusUnauthorized GTMSessionFetcherStatusUnauthorized
+  #define kGDataFetcherStatusForbidden GTMSessionFetcherStatusForbidden
```

Comment:
You seen you update our fork against the master so this change doesn't show up; as it has already landed.